packages/xrpc-server/tests/subscriptions.test.ts

```diff
@@ -99,12 +95,10 @@ describe('Subscriptions', () => {
     const countdown = Number(params.countdown ?? 0)
     for (let i = countdown; i >= 0; i--) {
       yield {
-        $type:
-          i % 2 === 0 ? 'io.example.stream2#even' : 'io.example.stream2#odd',
+        $type: i % 2 === 0 ? '#even' : '#odd',
         count: i,
       }
     }
-    yield { $type: 'io.example.stream2#done' }
```

Comment:
What should the behavior be for this?  I imagine it might end-up in the header discriminated as `#done`.

Reply:
yup that's right^^

altho this seemed strange since it was not in the schema 🤔 

I'm down to add something back in tho

Reply:
Agree, the same case doesn't really happen now that `codes` is gone.  I think it was a test of what happens when you yield a message with a type that doesn't have a matching code.